.github/workflows/pkg-pr-new-publish.yml

```diff
@@ -29,24 +29,24 @@ jobs:
           npm run build
       - name: Publish o1js on pkg-pr-new
         run: npx pkg-pr-new publish # Enable `--compact` once published to NPM with `repository` in package.json
-  pkg-pr-new-mina-signer:
-    strategy:
-      matrix:
-        node: [20]
-    runs-on: [ubuntu-latest]
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Setup Node.JS ${{ matrix.node }}
-        uses: actions/setup-node@v4
-        with:
-          node-version: ${{ matrix.node }}
-      - name: Build mina-signer
-        run: |
-          git submodule update --init --recursive
-          npm ci
-          cd src/mina-signer
-          npm ci
-          npm run prepublishOnly
-      - name: Publish mina-signer on pkg-pr-new
-        run: npx pkg-pr-new publish # Enable `--compact` once published to NPM with `repository` in package.json
+  # pkg-pr-new-mina-signer:
```

Comment:
Can you leave a comment on why this is commented out and any actions that should take place before enabling it?

Reply:
Actually let's try to release sequentially.

Reply:
Seems to work better, but for some reason it releases `o1js` instead of `mina-signer`
https://github.com/o1-labs/o1js/actions/runs/9722423299
I'll raise this issue with the tool maintainers.

Reply:
And final build with fixes https://github.com/o1-labs/o1js/actions/runs/9723324883